common/djangoapps/student/helpers.py

```diff
@@ -188,4 +193,8 @@ def check_verify_status_by_course(user, course_enrollment_pairs, all_course_mode
                     'verification_good_until': verification_good_until
                 }
 
+    if most_recent_verifications:
+        for key, value in status_by_course.iteritems():
+            status_by_course[key]['verification_good_until'] = max(most_recent_verifications).strftime("%m/%d/%Y")
```

Comment:
If we're going to set `verification_good_until` here, do we also need to set it on line 193?  If not, would it be possible to remove lines 187, 188 and 193?